vue-router-01/.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+// https://eslint.org/docs/user-guide/configuring
+
+module.exports = {
```

Comment:
The project name seems very generic. Please consider using specific project name.

Reply:
All required changes are fixed. Pls check